mm2src/mm2_p2p/src/behaviours/mod.rs

```diff
@@ -145,7 +145,7 @@ mod tests {
     }
 
     #[tokio::test]
-    #[cfg(not(windows))] // https://github.com/KomodoPlatform/atomicDEX-API/issues/1712
+    #[cfg(target_os = "linux")] // https://github.com/KomodoPlatform/atomicDEX-API/issues/1712
```

Comment:
Why only linux OS instead of UNIX platform?

Reply:
I guess it fails on MacOS too. Can we do "unix but not macos"?

Reply:
Thanks, will look to fix it in another PR. Will add this to the `nextest` issue